gcloud/logging/logger.py

```diff
@@ -291,10 +349,22 @@ def log_text(self, text, labels=None):
 
         :type labels: dict or :class:`NoneType`
         :param labels: (optional) mapping of labels for the entry.
+
+        :type insert_id: string or :class:`NoneType`
+        :param insert_id: (optional) unique ID for log entry.
+
+        :type severity: string or :class:`NoneType`
+        :param severity: (optional) severity of event being logged.
+
+        :type http_request: dict or :class:`NoneType`
+        :param http_request: (optional) info about HTTP request associated with
+                             the entry.
         """
-        self.entries.append(('text', text, labels))
+        self.entries.append(
+            ('text', text, labels, insert_id, severity, http_request))
```

Comment:
I thought about it, but it wouldn't help that much:  it would actually make *this* site *more* verbose.